### PR TITLE
Add spending_guidelines.md

### PR DESCRIPTION
**Pull request by kkongusa**
Add spending guideline document approved by the TSC.  Same as original except 'Transportation' replaced by 'Travel'. This change makes the language congruous with standard 'travel, meal, entertainment' expense type.

Signed-off-by: Keith Kong <kkongusa@gmail.com>

### CODE REVIEW

File: Policies/spending_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+- Traffic generators
+
+### Cables
+
+- Optical cables
+- Transceivers
+- DAC cables
+
+### Storage servers
```
Comment (glimchb) [inline Policies/spending_guidelines.md:33]: I think we can avoid buying storage servers and use software defined open source stacks for storage...

File: Policies/spending_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- Programmable switches
+
+### PDUs
+
+- PDUs
+- C13/C14/C15 power cables
+
+### KVMs
+
+- KVMs
```
Comment (glimchb) [inline Policies/spending_guidelines.md:48]: not sure is needed... all modern servers have built-in KVM... maybe for DPUs... for debug...

File: Policies/spending_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+used lab equipment generally available in the market are appropriate.
+Pre-production equipment or equipment that does not service the
+goals of the OPI lab are inappropriate.
+
+## Appropriate
+
+### General purpose servers
+
+- 1U, 2U, 3U, 4U rack servers with PCIE 3.0/4.0/5.0 slots
+- Pedestal servers
```
Comment (glimchb) [inline Policies/spending_guidelines.md:15]: I guess PXE, DNS< DHCP falls in those categories...

**Review by glimchb — APPROVED** [review]
few minor comments inside, but approving this PR

**Review by seroyer — APPROVED** [review]

**Comment by glimchb** [comment]
@kkongusa you can also add this new policy to the list here : https://github.com/opiproject/opiproject.org/pull/54

**Comment by kkongusa** [comment]
> @kkongusa you can also add this new policy to the list here : [opiproject/opiproject.org#54](https://github.com/opiproject/opiproject.org/pull/54)

Looks like this is already done: https://github.com/opiproject/opiproject.org/blob/main/content/policies/index.md

**Pull request merged**